src/python/pants/pantsd/service/scheduler_service.py

```diff
@@ -24,7 +25,14 @@ class SchedulerService(PantsService):
 
   QUEUE_SIZE = 64
 
-  def __init__(self, fs_event_service, legacy_graph_scheduler, build_root, invalidation_globs):
+  def __init__(
+    self,
+    fs_event_service,
+    legacy_graph_scheduler,
+    build_root,
+    invalidation_globs,
+    pidfile,
```

Comment:
Indicating that this is the `pantsd` pidfile, either in name or in a comment would be good.

Reply:
Done